nixos/tests/nix-ssh-serve.nix

```diff
@@ -18,22 +18,28 @@ in
          client.nix.package = pkgs.nix;
        };
      testScript = ''
-       startAll;
+       start_all()
 
-       $client->succeed("mkdir -m 700 /root/.ssh");
-       $client->copyFileFromHost("${ssh-config}", "/root/.ssh/config");
-       $client->succeed("cat ${snakeOilPrivateKey} > /root/.ssh/id_ecdsa");
-       $client->succeed("chmod 600 /root/.ssh/id_ecdsa");
+       client.succeed("mkdir -m 700 /root/.ssh")
+       client.succeed(
+           "cat ${ssh-config} > /root/.ssh/config"
+       )
+       client.succeed(
+           "cat ${snakeOilPrivateKey} > /root/.ssh/id_ecdsa"
+       )
+       client.succeed("chmod 600 /root/.ssh/id_ecdsa")
```

Comment:
Is there no way to do simple file operations on the node in the test driver with just python?

Reply:
There'll be https://github.com/NixOS/nixpkgs/issues/22401 soon to copy things generated in the VM to `$out`, but I don't think we can solve this one here much better currently.

We could indirect via a file in the nix store, and cp the store path, because it's also in the VM.

Reply:
There'll be https://github.com/NixOS/nixpkgs/issues/22401 soon to copy things generated in the VM to `$out`, but I don't think we can solve this one here much better currently.

We could indirect via a file in the nix store, and cp the store path, because it's also in the VM.